ROMFS/px4fmu_common/init.d-posix/1010_iris_opt_flow.post

```diff
@@ -1,3 +1,3 @@
 
 # shellcheck disable=SC2154
-mavlink stream -r 10 -s VISION_POSITION_ESTIMATE -u $udp_gcs_port_local
+mavlink stream -r 50 -s DISTANCE_SENSOR -u $udp_gcs_port_local
```

Comment:
Should this be ODOMETRY?

Reply:
Actually this was erroneous initially. We want to stream DISTANCE_SENSOR since opt flow model has a downward facing range sensor. We can add ODOMETRY as well though.